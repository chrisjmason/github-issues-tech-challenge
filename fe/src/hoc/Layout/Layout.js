import React from "react";

const Layout = (props) => (
  <div data-testid="layout">
    <header>this is header</header>
    <React.StrictMode>{props.children}</React.StrictMode>
    <footer>This is footer</footer>
  </div>
);

export default Layout;
