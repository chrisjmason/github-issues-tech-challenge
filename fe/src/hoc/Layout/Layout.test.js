import React from "react";
import { render, screen } from "@testing-library/react";
import Layout from "./Layout";

describe("Layout", () => {
  it("matches snapshot", () => {
    render(
      <Layout>
        <div>hello</div>
      </Layout>
    );
    expect(screen.getByTestId("layout")).toMatchSnapshot();
  });
});
