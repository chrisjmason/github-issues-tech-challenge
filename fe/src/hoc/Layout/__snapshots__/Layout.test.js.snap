// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout matches snapshot 1`] = `
<div
  data-testid="layout"
>
  <header>
    this is header
  </header>
  <div>
    hello
  </div>
  <footer>
    This is footer
  </footer>
</div>
`;
